reconcile/dashdotdb_slo.py

```diff
@@ -99,6 +99,13 @@ def _get_service_slo(self, slo_document: SLODocumentV1) -> list[ServiceSLO]:
         LOG.debug("SLO: processing %s", slo_document.name)
         result: list[ServiceSLO] = []
         for namespace_access in slo_document.namespaces:
+            # skip if there is a `targetNamespace` defined. in this case we
+            # don't know which is the prometheus that must evaluate the query.
+            # This functionlity is implemented here instead:
```

Comment:
```suggestion
            # This functionality is implemented here instead:
```

Reply:
I don't get this, this functionality is not implemented anywhere. We're not collecting for this kind of namespaces...

I honestly don't like the `targetNamespace` denomination for something that looks so RHOBS specific and that appears to be bound to how we process rules in that scenario...

Anyway, at least I'd just add a TODO here and a ticket...